src/Model/ModelManager.php

```diff
@@ -631,6 +631,11 @@ protected function camelize($property)
         return str_replace(' ', '', ucwords(str_replace('_', ' ', $property)));
     }
 
+    protected function generatePrefix(): string
```

Comment:
This name seems very generic in the context of a model manager. What is the purpose for the generated prefix?
IMO, we should improve the name and add a description if it is not clear enough.

Reply:
before it was hardcoded as `uniqid`, so I've extracted it into the separate method for testability